### PR TITLE
DOMA-2253 new layout

### PR DESCRIPTION
**Pull request by gegius**
[Design layouts](https://www.figma.com/file/P827yyueCLRT6o4gyNtzY6/CRM-(v1.doma.ai)?node-id=1606%3A0)

the button 'I am a resident' was not made and the header for phones was not completed

### CODE REVIEW

File: apps/condo/domains/common/components/HeaderActions.tsx [hunk 1]
```diff
@@ -138,3 +142,20 @@ export const ButtonHeaderActions: React.FC<IRightButtonHeaderActionsProps> = (pr
...
+
+export const TabsAuthAction: React.FC<ITabsActionsProps> = (props) => {
+    const { currentActiveKey } = props
+    const intl = useIntl()
+    const registerTab = intl.formatMessage({ id: 'pages.auth.RegistrationTitle' })
+    const signInTab = intl.formatMessage({ id: 'SignIn' })
+    return (
+        <Tabs
+            defaultActiveKey={currentActiveKey}
+            style={ { alignItems:'center', fontWeight: 500 } }
```
Comment (sitozzz) [inline apps/condo/domains/common/components/HeaderActions.tsx:154]: Move style to constant plz

File: apps/condo/domains/common/components/HeaderActions.tsx [hunk 2]
```diff
@@ -110,7 +114,7 @@ export const ButtonHeaderActions: React.FC<IRightButtonHeaderActionsProps> = (pr
     const { isMobile } = useContext(AuthLayoutContext)
     const { isSmall } = useLayoutContext()
     return (
-        <Row justify={'space-between'} gutter={[20, 0]}>
+        <Row justify={'space-between'} gutter={[5, 10]}>
```
Comment (sitozzz) [inline apps/condo/domains/common/components/HeaderActions.tsx:117]: Move gutter to constants.
`const IconContainerGutter: RowProps['gutter'] = [5, 10]`

File: apps/condo/domains/common/components/HeaderActions.tsx (hunk 1 above)
Comment (gegius) [reply apps/condo/domains/common/components/HeaderActions.tsx:154]: to move in constants styles with one style?

File: apps/condo/domains/common/components/HeaderActions.tsx (hunk 2 above)
Comment (gegius) [reply apps/condo/domains/common/components/HeaderActions.tsx]: Really it is necessary to transfer each gutter in constants? 🤔

Comment (sitozzz) [reply apps/condo/domains/common/components/HeaderActions.tsx]: Usually these values can be reused elsewhere on the same page

Comment (sitozzz) [reply apps/condo/domains/common/components/HeaderActions.tsx]: This approach has been proposed by @akarjakin. This is also due to how React works with the render tree under the hood

File: apps/condo/domains/user/components/containers/ResponsiveCol.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Col, ColProps } from 'antd'
+import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+
+export const ResponsiveCol = ({ style, children, ...otherProps }: ColProps) => {
+    const { isSmall } = useLayoutContext()
+    const modifiedStyle = {
+        ...style,
+        maxWidth: isSmall ? '600px' : '320px',
+    }
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/user/components/containers/ResponsiveCol.tsx:9]: add space after const block pls

File: apps/condo/pages/auth/change-password.tsx [hunk 4]
```diff
@@ -13,16 +13,16 @@ import { CHANGE_PASSWORD_WITH_TOKEN_MUTATION, GET_PHONE_BY_CONFIRM_PHONE_TOKEN_Q
 import { PASSWORD_IS_TOO_SHORT } from '@condo/domains/user/constants/errors'
 import { useAuth } from '@core/next/auth'
 import { BasicEmptyListView } from '@condo/domains/common/components/EmptyListView'
-import { ButtonHeaderActions } from '@condo/domains/common/components/HeaderActions'
 import { Loader } from '@condo/domains/common/components/Loader'
 import { useValidations } from '@condo/domains/common/hooks/useValidations'
 import { AuthLayoutContext } from '@condo/domains/user/components/containers/AuthLayoutContext'
 import { fontSizes } from '@condo/domains/common/constants/style'
+import { ResponsiveCol } from '../../domains/user/components/containers/ResponsiveCol'
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/change-password.tsx:20]: use domain imports pls

File: apps/condo/pages/auth/forgot.tsx [hunk 5]
```diff
@@ -11,19 +11,20 @@ import { WRONG_PHONE_ERROR, TOO_MANY_REQUESTS } from '@condo/domains/user/consta
...
 import { CountDownTimer } from '@condo/domains/common/components/CountDownTimer'
-import { ButtonHeaderActions } from '@condo/domains/common/components/HeaderActions'
 import { useValidations } from '@condo/domains/common/hooks/useValidations'
 import { PhoneInput } from '@condo/domains/common/components/PhoneInput'
 import { ValidatePhoneForm } from '@condo/domains/user/components/auth/ValidatePhoneForm'
 import { useGoogleReCaptcha } from 'react-google-recaptcha-v3'
 import { normalizePhone } from '@condo/domains/common/utils/phone'
 import { RegisterContext, RegisterContextProvider } from '@condo/domains/user/components/auth/RegisterContextProvider'
 import { Loader } from '@condo/domains/common/components/Loader'
+import { ResponsiveCol } from '../../domains/user/components/containers/ResponsiveCol'
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:21]: use domain import pls

File: apps/condo/pages/auth/forgot.tsx [hunk 6]
```diff
@@ -36,6 +37,10 @@ function ResetPageView () {
     const ExamplePhoneMsg = intl.formatMessage({ id: 'example.Phone' })
     const TooManyRequestsMsg = intl.formatMessage({ id: 'TooManyRequests' })
 
+    //TODO Set global style for ALL Phone Form Label(!!!!)
+    const REGISTER_PHONE_LABEL = <label style={ { alignSelf:'end' } }>{PhoneMsg}</label>
+
+
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:43]:
```suggestion
```

File: apps/condo/pages/auth/forgot.tsx [hunk 7]
```diff
@@ -36,6 +37,10 @@ function ResetPageView () {
     const ExamplePhoneMsg = intl.formatMessage({ id: 'example.Phone' })
     const TooManyRequestsMsg = intl.formatMessage({ id: 'TooManyRequests' })
 
+    //TODO Set global style for ALL Phone Form Label(!!!!)
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:40]: we write TODO in the following style: `TODO(DOMA-XXXX): description`

File: apps/condo/pages/auth/forgot.tsx [hunk 8]
```diff
@@ -111,64 +117,66 @@ function ResetPageView () {
...
+                                        rules={validations.phone}
+                                    >
+                                        <PhoneInput placeholder={ExamplePhoneMsg} />
+                                    </Form.Item>
+                                </Col>
+                                <Col span={24}>
+                                    <Form.Item>
+                                        <CountDownTimer action={startConfirmPhoneAction} id={'FORGOT_ACTION'} timeout={SMS_CODE_TTL}>
+                                            {({ countdown, runAction }) => {
+                                                const isCountDownActive = countdown > 0
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:153]: add space after const block

**Comment by AntonAL** [comment]
`UserWelcomeTitle` does not have margin `40px` from right-side, according to design.
Discuss overlapping problem with Max, maybe it is not critical for some browser widths.

File: apps/condo/domains/common/components/HeaderActions.tsx [hunk 9]
```diff
@@ -138,3 +149,21 @@ export const ButtonHeaderActions: React.FC<IRightButtonHeaderActionsProps> = (pr
...
+    const { currentActiveKey } = props
+    const intl = useIntl()
+    const registerTab = intl.formatMessage({ id: 'pages.auth.RegistrationTitle' })
+    const signInTab = intl.formatMessage({ id: 'SignIn' })
+
+    return (
+        <Tabs
+            defaultActiveKey={currentActiveKey}
+            style={TABS_STYLES}
+            onChange={(activeKey) => Router.push(activeKey)}
```
Comment (AntonAL) [inline apps/condo/domains/common/components/HeaderActions.tsx:163]:
```suggestion
            onChange={(activeKey) => Router.push(activeKey)}
            centered
```

File: apps/condo/domains/common/components/HeaderActions.tsx [hunk 10]
```diff
@@ -28,6 +28,12 @@ interface IRightButtonHeaderActionsProps {
     path: string
 }
 
+interface ITabsActionsProps {
+    currentActiveKey: string
+}
+
+const TABS_STYLES = { alignItems: 'center', fontWeight: 500 }
```
Comment (AntonAL) [inline apps/condo/domains/common/components/HeaderActions.tsx:35]:
```suggestion
const TABS_STYLES = { fontWeight: 500 }
```

**Comment by AntonAL** [comment]
There is a display issue of labels under Safari 15.2, which is not justified to bottom.

File: apps/condo/domains/user/components/containers/AuthLayout.tsx [hunk 11]
```diff
@@ -23,7 +23,7 @@ const AuthLayout: React.FC<IAuthLayoutProps> = (props) => {
     const intl = useIntl()
 
     const { children, ...otherProps } = props
-
+    console.debug('AuthLayoutPros', { ...otherProps })
```
Comment (AntonAL) [inline apps/condo/domains/user/components/containers/AuthLayout.tsx:26]:
```suggestion
```

**Review by sitozzz — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

**Pull request merged**